requirements-dev.txt

```diff
@@ -5,6 +5,7 @@ flaky>=3.3.0
 flake8==3.4.1
 hypothesis>=3.31.2
 py-geth==1.10.2
+py-solc==2.1.0
```

Comment:
Not clear why this dependency was added?

Reply:
I added this so that the "quickstart" example in the README.md could be run.  It's purely a developer convenience, so they don't have to install this package to work with solidity code in python.  Can remove it if you don't want it in there.